android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -932,4 +942,57 @@ public String dataChannelStateString(DataChannel.State dataChannelState) {
         }
         return null;
     }
+
+    class DataChannelObserver implements DataChannel.Observer {
```

Comment:
This comment is not a call for action. I really want to hear your opinion on splitting this big file into smaller ones. I suppose we wouldn't start with their source code but I feel like our source code stands a greater chance of being split out of this file. I'm aware that consistency is at play here as well but... Anyway, I'd like to hear your thoughts, @paweldomas.